src/applications/check-in/day-of/pages/NextOfKin.jsx

```diff
@@ -80,9 +79,7 @@ const NextOfKin = props => {
   } else {
     return (
       <>
-        {(isUpdatePageEnabled || isDemographicsPageEnabled) && (
-          <BackButton router={router} />
-        )}
+        {isUpdatePageEnabled && <BackButton router={router} />}
```

Comment:
Like the EmergencyContact page, I think this one should also always have a back button